recipes/vclust/run_test.sh

```
@@ -0,0 +1,8 @@
+# !/bin/bash
```

Shellcheck

[error] Remove spaces between # and ! in the shebang.

(SC1115)